.github/workflows/release.yml

```diff
@@ -3,14 +3,35 @@ name: release
 on:
   push:
     branches:
-      - release/*
+      - release
 
 jobs:
+  check_branch:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check branch
+        if: github.event_name == 'workflow_dispatch' && github.ref != 'refs/heads/release'
+        run: |
+          echo "ERROR: You can only run this on release branches"
+          exit 1
   tests:
     uses: ./.github/workflows/test.yml
+  get_version_number:
+    runs-on: ubuntu-latest
+    outputs:
+      version_string: ${{ steps.get-latest-release.outputs.release }}
+    steps:
+      - uses: pozetroninc/github-action-get-latest-release@master
+        id: get-latest-release
+        with:
+          repository: ${{ github.repository }}
+      - name: Echo Version
+        run: |
+          echo "${{ steps.get-latest-release.outputs.release }}"
```

Comment:
I don't think you can really merge in the new workflow stuff without any of it being documented, could you put something brief in the readme please?